.github/actions/build-essential/action.yml

```diff
@@ -1,7 +1,7 @@
-name: 'Build essential packages'
-description: 'Build essential packages'
+name: Build essential packages
+description: Build essential packages
 runs:
-  using: 'composite'
+  using: composite
   steps:
     - name: Build essential packages used by others
       shell: bash
```

Comment:
is bash an option on windows?

Reply:
I guess so since it passed, however I don't think it has much utility here so we could remove it.